package Protos;

import "base.proto";

message SynapseInfo {
    required uint32 idPre = 1;
    required double dendriteDelay = 2;
    required double weight = 3;
    required double potential = 4;
}

